2d/benchmarks/dambreak_Colagrossi/helpers.py

```diff
@@ -1,12 +1,14 @@
 def CreateFig():
     from tables import  openFile
-    archive = openFile('dambreak_Colagrossi0.h5','r')
+    archive = openFile('dambreak_Colagrossi_p.h5','r')
     import dambreak_Colagrossi
     import dambreak_Colagrossi_so
     import matplotlib.tri as mtri
     from matplotlib import pyplot as  plt
     import numpy as np
     domain = dambreak_Colagrossi.domain
+    domain.L = dambreak_Colagrossi.tank_dim
```

Comment:
@tridelat the L and x data members of domains represent the lower left hand corner and extents of the bounding box in general. The new spatial tools based workflows  don't seem to be setting them, but it would be nice if they did. You basically have to search the vertices for the max and min in each dimension.